FV2310/IFTSTA/conditions.json

```diff
@@ -459,6 +459,16 @@
     "condition_text": "Wenn mehr als ein Grund vorliegt und die voranstehenden DE9013 dieses STS nicht ausreichen",
     "edifact_format": "IFTSTA"
   },
+  {
+    "condition_key": "144",
+    "condition_text": "Wenn RFF+Z45 in dieser SG15 nicht vorhanden",
+    "edifact_format": "IFTSTA"
+  },
+  {
+    "condition_key": "145",
+    "condition_text": "Wenn RFF+Z17 in dieser SG15 nicht vorhanden",
+    "edifact_format": "IFTSTA"
+  },
```

Comment:
Correct as in https://github.com/Hochfrequenz/edi_energy_mirror/blob/master/edi_energy_de/FV2310/IFTSTAAHB2.0eKonsolidierteLesefassungmitFehlerkorrekturenStand11.03.2024_99991231_20240311.pdf (page 40)